recipes/wk_spectra/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "wk_spectra" %}
+{% set version = "0.1.1" %}
+
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d217783761ae98ef5f2bfc83376d61bcb9d9e59cfbf650e29e21f0a986c996c9
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv "
+
+requirements:
+  host:
+    - coverage
+    - matplotlib >=3.0.0
+    - netcdf4 >=1.4.1
+    - nose
+    - numpy >=1.15.2
+    - pandas >=0.23.4
+    - pip
+    - python
+    - scipy >=1.1.0
+    - seaborn ==0.9.0
+    - sphinx
+    - sphinx_rtd_theme
+  run:
+    - coverage
+    - matplotlib >=3.0.0
+    - netcdf4 >=1.4.1
+    - nose
+    - numpy >=1.15.2
+    - pandas >=0.23.4
+    - python
+    - scipy >=1.1.0
+    - seaborn ==0.9.0
+    - sphinx
+    - sphinx_rtd_theme
+
+test:
+  imports:
+    - wk_spectra
+
+about:
+  home: https://github.com/ajaramillomoreno/wk_spectra
+  license: BSD
+  license_family: BSD
+  license_file:
+  summary: A Python package for the construction of the Wheeler-Kiladis Space-Time Spectra.
+  doc_url:
```

Comment:
The documentation link upstream in the README is broken. If there is a documentation site, please include it here. If not, you can remove the `doc_url` line.